client/core/trade.go

```diff
@@ -567,7 +569,7 @@ func (t *trackedTrade) swapMatches(matches []*matchTracker) error {
 	for _, coinID := range coinIDs {
 		coin, found := t.coins[hex.EncodeToString(coinID)]
 		if !found {
-			return errs.add("coin ID %x not found", coinID)
+			return errs.add("coin ID %s not found", coinID)
```

Comment:
I know this is a random fix, but how would you feel about the following instead:

```go
assetID := t.wallets.fromAsset.ID
coinStr, err := asset.DecodeCoinID(assetID, coinID)
	if err != nil {
	log.Warnf("invalid coin for asset %d: %v", assetID, err)
	coinStr = coinID.String()
}
return errs.add("coin ID %s not found", coinStr)
```

The undecoded coinID bytes just bug me.